micro-rdk/src/common/robot.rs

```diff
@@ -106,6 +113,7 @@ pub struct LocalRobot {
     // is NOT a valid timestamp. For actual timestamps, the real time should be set on the system
     // at some point using settimeofday (or something equivalent) and referenced thereof.
     pub(crate) start_time: Instant,
+    cloud_metadata: Option<CloudMetadata>,
```

Comment:
even if it's somewhat incorrect I think we can avoid the Option here. In can't see and case where we wouldn't be able to populate CloudMetadata from the config. And if a robot is constructed by default then we should be in provisioning

Reply:
Won't we be able to serve on an isolated network without cloud metadata?

Reply:
Scratch that previous question, my actual concern is this: the cloud meta data is an option in the config proto itself, so by leaving it as an option I hope to stop a potential error in the config endpoint that should be inconsequential from breaking the ability to create a robot from the config